content/docs/reference/pkg/aws/ec2/getvpcendpoint.md

```diff
@@ -991,7 +991,7 @@ The following output properties are available:
 ## Supporting Types
 
 
-<h4 id="getvpcendpointdnsentry">Get<wbr>Vpc<wbr>Endpoint<wbr>Dns<wbr>Entry</h4>
+<h4 id="getvpcendpointdnsentry">get<wbr>Vpc<wbr>Endpoint<wbr>Dns<wbr>Entry</h4>
```

Comment:
Shouldn't these remain title case?

Reply:
Yeah you are right. Let me look at it. I also need to rebase with `master` in this repo and regen all docs here.

Reply:
The title is fixed now.